src/boot/efi/measure.c

```diff
@@ -123,6 +124,68 @@ static EFI_STATUS tpm2_measure_to_pcr_and_event_log(
                         tcg_event);
 }
 
+static EFI_STATUS cc_measure_to_mr_and_event_log(
+                EFI_CC_MEASUREMENT_PROTOCOL *cc,
+                uint32_t pcrindex,
+                EFI_PHYSICAL_ADDRESS buffer,
+                uint64_t buffer_size,
+                const char16_t *description)
+{
+        _cleanup_free_ EFI_CC_EVENT *event;
+        uint32_t mr;
+        EFI_STATUS err;
+        size_t desc_len;
+
+        assert(description);
+        assert(cc);
+
+        desc_len = strsize16(description);
+        event = xmalloc(offsetof(EFI_CC_EVENT, Event) + desc_len);
+        memset(event, 0, offsetof(EFI_CC_EVENT, Event) + desc_len);
+
+        err = cc->MapPcrToMrIndex(cc, pcrindex, &mr);
```

Comment:
could you add the explanations from the commit msg here? this is relevant info that anyone looking at the sources should see

Reply:
The mapping is vendor specific. The commit message states status quo with Intel TDX but it may not be a good fit here. I added a comment describing the meaning of this call.